docs/index.md

```diff
@@ -0,0 +1,98 @@
+## Overview
+This guide walk-through the process of deploying an Sample Java application in CFT.
+
+This guide uses a pre-configured application from a template. Application is deployed to a kubernetes cluster. 
+At the end of this tutorial, you should be able to access an working application via VPN and made changes to it.
+
+## Prerequisite
+
+`Github` access is required to complete the steps in this tutorial. Following the [link](https://hmcts.github.io/onboarding/team/github.html#github) to request access.
+
+## High Level Steps
+
+- Create a Repository and build pipeline
+- Build application
+- Configure load balancing for HA
+- Deploy application
+- Access application
+- Customize application
+- Troubleshooting
+
+## Steps
+
+#### Create a Repository and build pipeline
+
+1. Access [Backstage](https://backstage.platform.hmcts.net/create) to create a new github repo by selecting predefined [`Spring Boot Service`](https://backstage.platform.hmcts.net/create/templates/springboot-template) template. Please make sure name of  Github repo  starts with `labs*`. This enables Jenkins build tool to organize the folder based on Github repo names.
```

Comment:
```suggestion
1. Click `Create` in the Backstage sidebar and select [`Spring Boot Service`](https://backstage.platform.hmcts.net/create/templates/springboot-template) template. Prefix your GitHub repository name with `labs-*`. This enables Jenkins to automatically pickup the folder based on the GitHub repository name.
```

Reply:
@timja [ url ](https://backstage.platform.hmcts.net/create/templates/springboot-template)) takes us directly to landing page for the template.